src/backend/joanie/core/filters/admin.py

```diff
@@ -290,3 +290,41 @@ def filter_by_query(self, queryset, _name, value):
             | Q(owner__first_name__icontains=value)
             | Q(owner__last_name__icontains=value)
         ).distinct()
+
+
+class EnrollmentAdminFilterSet(filters.FilterSet):
```

Comment:
Same here ? Start a new module by filterset ?

Reply:
I did it for `filterset` and `viewset`. But about serializers, by experience, it's a little bit trickier due to circular import issues.